src/utils/versions/getVersionsColors.ts

```diff
@@ -25,7 +25,7 @@ export const COLORS = [
     '#b22222', // firebrick
 ];
 
-export const GRAY_COLOR = '#bfbfbf'; // --yc-color-base-neutral-hover
```

Comment:
This color is no longer corresponds to uikit hover color, but it's left unchanged